godotproject/Player.gd

```diff
@@ -406,9 +407,26 @@ func _physics_process(delta):
 			velocity_after_swing += velocity.normalized() * tongue_exit_jump_bonus_speed_in_dirn
 			velocity_after_swing += Vector2.UP * tongue_exit_jump_bonus_speed_up
 			velocity = velocity_after_swing
-		elif tongue_pressed:
-			# When we exit swinging using the tongue button, just drop them out of swinging
+		
+		if tongue_pressed:
+			# When we exit swinging using the tongue button, shoot player in direction of tongue
```

Comment:
This should be an elif, just to avoid having to think about what would happen if they pressed jump and tongue on the same frame.
```suggestion
		
		elif tongue_pressed:
			# When we exit swinging using the tongue button, shoot player in direction of tongue
```